### DISCUSSION TITLE
Is it normal that star rating rounds up?

### DISCUSSION THREAD

**Discussion by Machinedra** [ppy/osu#17837] [Q&A]
Star rating rounding up makes the game confusing for a lot of player, it's strange that it does it, making you not getting a 6* pass or fc because it's a 5.99999
(ex: https://osu.ppy.sh/beatmapsets/432290#osu/932185 +DT or https://osu.ppy.sh/beatmapsets/1073074#osu/2250747 NM)

**Comment by bdach** [comment]
Everywhere where star ratings are displayed in lazer they are rounded to 2 decimal places.

The only time where this would realistically matter is with pass achievements. But those are not implemented in lazer yet, and if stable is failing to award those due to rounding, then that is arguably a bug (which should be fixed when lazer starts awarding achievements).

**Reply by Machinedra** [reply]
it's mostly about pass/fc medals that i asked that question, thanks for answering!

**Comment by smoogipoo** [comment]
I think it's already come up internally that we're storing SR in the database (or was it PP?) to a higher precision than necessary. Could consider either rounding the output to the database, or rounding in the medal-awarding code.
